modules/RTC/RTCUtils.js

```diff
@@ -869,17 +869,41 @@ class RTCUtils extends Listenable {
                 this.p2pPcConstraints
                     = JSON.parse(JSON.stringify(this.pcConstraints));
 
-                // Allows sending of video to be suspended if the bandwidth
-                // estimation is too low.
-                if (!options.disableSuspendVideo) {
-                    this.pcConstraints.optional.push(
-                        { googSuspendBelowMinBitrate: true });
+                const {
+                    forceSuspendVideo,
+                    disableSuspendVideo
+                } = options;
+                const suspendVideoEnabled
+                    = typeof forceSuspendVideo === 'undefined'
+                        || (typeof forceSuspendVideo === 'number'
+                            && Math.random() < forceSuspendVideo);
+                const disableSuspendVideoForJVBPC
+                    = Boolean(disableSuspendVideo);
+
+                if (typeof forceSuspendVideo === 'number'
+                    && suspendVideoEnabled) {
+                    logger.info(`Enable suspend video mode (ratio=${
+                        forceSuspendVideo}, config.disableSuspendVideo=${
+                        disableSuspendVideoForJVBPC.toString()}`);
+                    Statistics.analytics.addPermanentProperties({
+                        suspendVideoEnabled: true,
+                        disableSuspendVideoForJVBPC
+                    });
                 }
 
-                // There's no reason not to use this for p2p
-                this.p2pPcConstraints.optional.push({
-                    googSuspendBelowMinBitrate: true
-                });
+                if (suspendVideoEnabled) {
+                    if (!disableSuspendVideoForJVBPC) {
```

Comment:
If we use the "force" property we get the following behavior:
If it succeeds (Math.random() < forceSuspendVideo), then we add the suspend constraint for p2p and honor the existing configuration for the jvb pc.
If it doesn't succeed, we ignore the configuration and do not add the constraint to either PC. This seems strangely inconsistent. 

Can we just add a second flag for p2p, and then have the "force" property work as an actual force (i.e. override the config in both cases)? 

Or, in case what we want to implement is a forced override only for p2p, then add a second flag for p2p and have the "force" property only override the p2p?

Reply:
My understanding about the task is the following:
We want to implement a way to enable the suspend video parameter for 1/2 of the users and disable it for the other half. I have the impression that this parameter will be temporary, only for internal use. @emcho please correct me if I'm wrong.

I'm sorry that I haven't added this to the description to avoid confusion.

I wouldn't say that the "strangely inconsistent" behavior is introduced by the  `forceSuspendVideo` option (maybe only the naming is a little "strangely inconsistent" and the lack of comments :) ). The situation before the PR is the following:
We have config option `disableSuspendVideo` that will disable the suspend video only for the JVB peer connection but in the same time the p2p peer connection will always use the suspend video. I think this is the cause for the inconsistent behavior.

Basically my implementation is preserving the previous logic and makes sure that if somebody is using a config without `forceSuspendVideo` the behavior will be exactly the same as before.

I agree that if we keep the 'force' naming it make sense to override `disableSuspendVideo` option. Although both of them are specified in the config and I don't think it actually matters from technical perspective or especially if we are going to remove this option soon.

@bgrozev I think you have a preference for fixing the situation by adding one more option for the p2p peer connection, right? Isn't this too complicated and hard to document? What would be the use case where somebody will want to enable suspend video only for one of the peer connections? I'm also worried that if we introduce new option for disabling the p2p peer connection, we would want to change the `disableSuspendVideo` option naming which will break the backward compatibility. My proposal is to use `disableSuspendVideo` option for all peer connections. WDYT?

Reply:
> We want to implement a way to enable the suspend video parameter for 1/2 of the users 
> and disable it for the other half.

is this for both peer connections (p2p and bridge) or only the bridge?

Reply:
My understanding is for both. I can't think of a reason why would we split them. @emcho could you please confirm?

Reply:
We discussed this in person. Let's wait for Hristo's new version and comment on that.